src/client/components/inputs/index.tsx

```diff
@@ -222,29 +223,32 @@ export const RawInputField: React.FC<React.InputHTMLAttributes<
 > & {
   label?: string
   errors?: string
+  helperText?: string
   variant?: variantType
   showErrorIcon?: boolean
+  disabled?: boolean
 }> = ({
   errors,
+  disabled,
   showErrorIcon = false,
+  helperText,
   label,
   className,
   variant = 'light',
   ...props
 }) => (
   <InputFieldContainer className={className}>
-    <Wrapper errors={errors} variant={variant}>
-      <TextWrapper>
-        {label && <Label htmlFor={props.id}>{label}</Label>}
-        <StyledRawInput {...props} />
-      </TextWrapper>
-      <SymbolWrapper>
-        {(errors || showErrorIcon) && (
-          <WarningIcon color={inputVariants[variant].error} />
-        )}
-      </SymbolWrapper>
+    {label && <Label htmlFor={props.id}>{label}</Label>}
+    <Wrapper errors={errors} variant={variant} disabled={disabled}>
+      <StyledRawInput disabled={disabled} {...props} />
     </Wrapper>
-    {errors && <ErrorText variant={variant}>{errors}</ErrorText>}
+    {helperText && <HelperText>{helperText}</HelperText>}
+    {errors && (
+      <ErrorText>
+        <WarningTriangle size={13} />
+        {errors}
+      </ErrorText>
+    )}
```

Comment:
From figma it looks like the error replaces the helper text - or have you discussed something else with Zak?

Reply:
yeah not sure about this one yet. Asked Zak in Figma but he is in vacation so still waiting on a reply for this.

Reply:
true, we should probably not display the helper text and error message